Data/models/Bookmark.swift

```diff
@@ -243,13 +243,25 @@ public final class Bookmark: NSManagedObject, WebsitePresentable, Syncable, CRUD
             format: "%K == %@ AND %K == NO", parentFolderKP, parent ?? nilArgumentForPredicate, isFavoriteKP)
     }
     
+    public class func add(defaults: [(url: URL, title: String)]) {
```

Comment:
This doesn't need to be specific to `defaults`, so just make it a general function `add(bookmarkData)` or something. But regardless, probably not needed for this PR.

Reply:
renamed